src/components/CreatePost/CreatePost.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+
+import { kwitterApiClient } from '../../api/client';
+import { KweetsList } from '../../components/KweetsList';
+import { Kweet as KweetModel } from '../../model/kweet';
+import { ListKweetsResponse } from '../../model/kwitterApiModels';
+import {
+  KwitterButton,
+  KwitterPost,
+  KwitterTextArea,
+  ProfilePhoto
+} from './styled';
+
+export const CreatePost = (): React.ReactElement => {
+  const [post, setPost] = useState<string>('');
+  const [kweetList, setKweetList] = useState<KweetModel[]>([]);
```

Comment:
Change request: I would just use `ListKweetsResponse` type here as well for consistency.